packages/caravan-fees/src/cpfp.ts

```diff
@@ -0,0 +1,105 @@
+import { Transaction } from "bitcoinjs-lib-v5";
+import { unsignedMultisigTransaction } from "@caravan/bitcoin";
+import BigNumber from "bignumber.js";
+import { CPFPOptions, UTXO } from "./types";
+import { estimateVirtualSize } from "./utils";
+
+/**
+ * Prepare a CPFP (Child-Pays-for-Parent) transaction.
+ * This function doesn't broadcast the transaction, it just prepares it.
+ * @param options CPFP options including the parent transaction and new fee rate
+ * @returns A new transaction ready for signing and broadcasting
+ */
+export function prepareCPFPTransaction(options: CPFPOptions): Transaction {
+  const {
+    parentTransaction,
+    newFeeRate,
+    availableUTXOs,
+    destinationAddress,
+    network,
+    multisigDetails,
+  } = options;
```

Comment:
Why not destructure  in the function sig? 
```suggestion
export function prepareCPFPTransaction({
    parentTransaction,
    newFeeRate,
    availableUTXOs,
    destinationAddress,
    network,
    multisigDetails,
  }: CPFPOptions): Transaction {

```

Reply:
Sorry that's a bad habit I got from writing express code ..